modules/services/avizo.nix

```diff
@@ -0,0 +1,71 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let cfg = config.services.avizo;
+in {
+
+  meta.maintainers = [ hm.maintainers.pltanton ];
+
+  options.services.avizo = {
+    enable = mkEnableOption ''
+      Whether to enable avizo.
+      Avizo is a simple notification daemon, mainly intended to be used for multimedia keys for example with Sway.
+    '';
+    config = mkOption {
```

Comment:
This should probably be called `settings` for consistency (and because `config` is quite overloaded)

Reply:
Yeah, definitely need to change the name to `settings`.